### ISSUE TITLE
More Developed graph of the learning rate finder

### ISSUE THREAD

**Issue by BttMA** [PyTorchLightning/pytorch-lightning#11701]
## 🚀 Feature

hello :)
I've been using the `lr_finder` for a while and I can tell it is quite a helper specially when you training a complex model with more data. but, when you advance more and more in the training process, a good learning rate become more and more difficult to find as you may skip the global minima which is no good for the model. In the graph that we can have by `lr_finder.plot(suggest=True)`, we can have only one possibility of a good learning rate for start like this one ... 
![Screenshot 2022-02-02 at 12 59 00 PM](https://user-images.githubusercontent.com/90634663/152149646-875155eb-71e3-4680-93ee-1910db0d7fbe.png)

What I am suggesting is to add more arguments to the `lr_finder.plot` so we can see more values of the learning rate.

### Motivation

with this new feature (if we can call it that), we can have more efficient way to search for a good learning rate.

### Pitch
you can put somming like this 

`lr_finder.plot(suggest=True, sug_lr=1.5e-05) #like this we have the suggested value also the value that you gave the sug_lr argument`

cc @borda @akihironitta @rohitgr7

**Comment by justusschock** [comment]
hey @BttMA , Thanks for the issue. However, I am afraid, that I am not so much in favor of it. As you can see [here](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/tuner/lr_finder.py#L163), the `plot` function just returns a matplotlib figure, on which you could plot additional stuff (e.g. by accessing `results["loss"]` of the lr finder).

Why I am not so much in favour of your proposal for automating this is the following:
When you suggest an lr manually to plot it, you don't know if this specific lr was actually tested (most likely not) and therefore you don't have a corresponding loss value to plot. Of course you could do some search for the nearest neighbor, but IMO that isn't exactly what you wand and also can yield undesired results.

Thoughts @SkafteNicki ?

**Comment by BttMA** [comment]
Hello, @justusschock :)
Thank you for your quick replay.

Maybe you are right but the general concept behind the `lr_finder` is to test different lr on a small amount of data (100 by default) and it doesn't give you an exact value. It is just a suggestion for a start value. It is not well defined. It may be good but it may be not too. Only with some tests, you can decide :)

Also, you can see in the above graph between 10^-8 and 10^-7 that we have a curve. It may be a global optimum but the lr finder skip it and we (maybe _I_ dont know haha!) dont know why? but what I suggest can be an option as we can see the lr on the graph and see where it will be.

Thanks!
